Lib/http/client.py

```diff
@@ -1363,7 +1366,8 @@ def getresponse(self):
             except ConnectionError:
                 self.close()
                 raise
-            assert response.will_close != _UNKNOWN
+            if self.chunked == _UNKNOWN:
```

Comment:
I feel that this behavior is different than the original code's
> assert response.will_close != _UNKNOWN

Furthermore, I'm unsure if `HTTPException`, is the correct exception to use here; the documentation states that ConnectionError is usually raised by `getresponse`

I would suggest instead:
``` python
if response.will_close == _UNKNOWN:
    raise ConnectionError("Unknown if response will close.")
```
Thanks for the contribution!

Reply:
Oh, you're completely right. I took the three as equals and missed the mark. Thanks for the review!